### PR TITLE
[T4A4][W10-B4] Karen Ang

### PR DESCRIPTION
**Pull request by karen**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -83,5 +83,18 @@ public int hashCode() {
...
+    
+    public String getPrintableString() {
+    	return getPrintableString(name, phone, email, address);
+    }
+    
+    public String getPrintableString(Printable...printables){
+    	StringBuilder builder = new StringBuilder();
+    	for (Printable p : printables){
+    		builder.append(p.getPrintableString());
+    		builder.append(" ");
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Person.java:95]: Doesn't this result in an extra space at the end? You could have used `StringJoiner` or `String.join` instead.

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Printable.java:3]: Missing header comment. All non trivial classes/methods should have javadoc format header comments.

**Comment by damithc** [comment]
Some comments added. Please close the PR after reading comments.

**Comment by karen** [comment]
Noted, thank you.

**Pull request closed without merging**